network/retrieval/retrieve_test.go

```diff
@@ -621,7 +621,7 @@ func newRetrievalTester(t *testing.T, prvkey *ecdsa.PrivateKey, netStore *storag
 		prvkey = key
 	}
 
-	r := New(kad, netStore, kad.BaseAddr(), nil)
+	r := New(kad, netStore, network.NewBzzAddr(kad.BaseAddr(), network.PrivateKeyToBzzKey(prvkey)), nil)
```

Comment:
PrivateKeyToBzzKey is creating an Overlay address, but here is used in place of Underlay address. Is it correct to set it to nil as it is not available in this function and not used in these tests?

Reply:
Yup, I think we can set it to nil as well.